library/control/src/modules/ProductControl.rb

```diff
@@ -1336,16 +1337,47 @@ def RunFrom(from, allow_back)
           end
         end
 
-        Hooks.run("before_#{step_name}")
+        client_name = getClientName(step_name, step_execute)
 
-        result = WFM.CallFunction(getClientName(step_name, step_execute), args)
+        # Check if client exist before continuing
+        if WFM.ClientExists(client_name)
+          Hooks.run("before_#{step_name}")
 
-        # this code will be triggered before the red pop window appears on the user's screen
-        Hooks.run("installation_failure") if result == false
+          result = WFM.CallFunction(client_name, args)
 
-        result = Convert.to_symbol(result)
+          # This code will be triggered before the red pop window appears on the user's screen
+          Hooks.run("installation_failure") if result == false
 
-        Hooks.run("after_#{step_name}")
+          result = result.to_sym
+
+          Hooks.run("after_#{step_name}")
+        else
+          # Client not found. Ask the user if want to continue (related to bsc#1180954)
+          Builtins.y2error("Client '%1' not found", client_name)
+
+          text = format(
+            # TRANSLATORS: Message warning the user that a client is missing where %{client} is
+            # replaced by the client name (e.g. "registration", "user")
+            _(
+              "Something went wrong and the expected '%{client}' dialog was not found.\n\n" \
+              "Would you like to continue anyway?"
```

Comment:
I would say `Would you like to skip dialog and continue anyway?`

Reply:
@jreidinger : I am not sure if that makes the user's decision any easier; it just complicates the question that we are asking. Remember that the user has to answer "Yes" or "No" here. Adding the aspect of skipping the dialog would make users want to get more options to choose from; but we have more options. Let's keep this simple.

And besides, it would of course be "skip **the** dialog". ;-)